README.md

```diff
@@ -39,6 +41,22 @@ The accelerator adapts prompts based on the model type for enhanced performance.
 
 Please [see this video](https://aka.ms/InfoAssist/video) for use cases that may be achievable with this accelerator.
 
+# Response Generation Approaches
+
+## Work(Grounded)
+It utilizes a retrieval-augmented generation (RAG) pattern to generate responses grounded in specific data sourced from your own dataset. By combining retrieval of relevant information with generative capabilities, It can produce responses that are not only contextually relevant but also grounded in verified data. The RAG pipeline accesses your dataset to retrieve relevant information before generating responses, ensuring accuracy and reliability. Additionally, each response includes a citation to the document chunk from which the answer is derived, providing transparency and allowing users to verify the source. This approach is particularly advantageous in domains where precision and factuality are paramount. Users can trust that the responses generated are based on reliable data sources, enhancing the credibility and usefulness of the application.Specific information on our Grounded (RAG) can be found in [RAG](docs/features/cognitive_search.md#azure-ai-search-integration)
+
+## Ungrounded
+It leverages the capabilities of a large language model (LLM) to generate responses in an ungrounded manner, without relying on external data sources or retrieval-augmented generation techniques. The LLM has been trained on a vast corpus of text data, enabling it to generate coherent and contextually relevant responses solely based on the input provided. This approach allows for open-ended and creative generation, making it suitable for tasks such as ideation, brainstorming, and exploring hypothetical scenarios. It's important to note that the generated responses are not grounded in specific factual data and should be evaluated critically, especially in domains where accuracy and verifiability are paramount.
+
+## Work and Web ( Grounded and Web)
+It offers two response options: one generated through our retrieval-augmented generation (RAG) pipeline, and the other obtained directly from the web. When users opt for the RAG response, they receive a grounded answer sourced from your data, complete with citations to document chunks for transparency and verification. Conversely, selecting the web response provides access to a broader range of sources, potentially offering more diverse perspectives. Each web response is accompanied by citations of web links, allowing users to explore the original sources for further context and validation. Upon request, It can also generate a final response that compares and contrasts both responses. This comparative analysis allows users to make informed decisions based on the reliability, relevance, and context of the information provided.
```

Comment:
Is Web ungrounded? I believe it's still grounded, but via Web results? In Transparency I'm addressing these as four separate items: Internally-grounded, Externally-grounded, Ungrounded, and Compare Internally- to Externally-grounded.

Reply:
Yes Web is grounded. Updated content. For Term I used terms that matches with UI not to confuse users.